homeworks/vlad.makedon_Vladmaked/homework_7/fibonacci.js

```diff
@@ -0,0 +1,10 @@
+function getFibonacciNumbers(rand) {
+    const arrayOfFibonacciNumbers = [0, 1];
+    for (let i = 2; arrayOfFibonacciNumbers[i - 1] + arrayOfFibonacciNumbers[i - 2] < rand; i++) {
+        const currentNumber = arrayOfFibonacciNumbers[i - 1] + arrayOfFibonacciNumbers[i - 2];
+        arrayOfFibonacciNumbers.push(currentNumber);
+    }
+    return arrayOfFibonacciNumbers;
+}
```

Comment:
also, if you assume that your function `getFibonacciNumbers(rand)` may be called without arguments (as you call it in tests), good practice is to provide some default value for the argument if it is not provided explicitly